crates/messages/src/lib.rs

```diff
@@ -437,38 +437,21 @@ pub enum Resources {
 }
 
 #[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct Status {
+    pub round: u32,
+    pub metric_name: String,
+    pub value: f32,
```

Comment:
How about we already add support for multiple metrics by using a map here?

```suggestion
    pub metrics: HashMap<String, f32>,
```

Otherwise, as we're currently only report the loss, we could remove `metric_name` for now and rename `value` to `loss`.